docs/application/dotnet/guides/multimedia/camera.md

```diff
@@ -397,7 +395,7 @@ To set some attributes:
     camera.Settings.Brightness = brightnessRange.Min;
     ```
 
-## Releasing Resources
+## Releasing resources
```

Comment:
Gerund forms should be avoided for headings.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#verb-form
```suggestion
## Release resources
```